### DISCUSSION TITLE
`Missing semicolon` on v4 upgrade

### DISCUSSION THREAD

**Discussion by mobilutz** [fnando/i18n-js#684] [Q&A]
Hi,

this might not be related to i18n-js itself, but more to general webpack config, but as I only see the error during the v4 upgrade, I will ask it here.

After following the [Migration Guide](https://github.com/fnando/i18n-js/blob/main/MIGRATING_FROM_V3_TO_V4.md) and changing code to use the new implementation, I cannot make real content in the `translation.json` file work. Even when I put just `{ "de": {} }` inside the json, I get the following error:
```
Compiled with problems:

ERROR in ./webpack/i18n/translations.json

Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /Users/ll/code/appapp/webpack/i18n/translations.json: Missing semicolon. (2:6)

  1 | {
> 2 |   "de": {
...
```

I am 99% certrain, that this has to do with how our 10-year old monolithic application is generating something through webpack.

Maybe you can point me/us to the right direction.

Thanks

**Comment by fnando** [comment]
If I had to guess, I think your webpack configuration is treating JSON files as JavaScript files, so that'd be my first recommendation: check your config.
